Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationPage.xaml

```diff
@@ -208,11 +208,11 @@
                             Orientation="Horizontal">
                     <TextBlock FontSize="18"
                                FontWeight="Bold"
-                               Text="Launch count:"
+                               Text="Total launch count:"
                                TextWrapping="Wrap" />
                     <TextBlock Margin="20,0"
                                FontSize="18"
-                               Text="{x:Bind LaunchCount}"
+                               Text="{x:Bind TotalLaunchCount}"
```

Comment:
We should show both in the sample and add a button which lets the user reset it to see how it behaves.

Reply:
Make senses. Will do it.